build.gradle

```diff
@@ -1196,7 +1196,8 @@ project(':connect:runtime') {
 
   dependencies {
     compile project(':connect:api')
-    compile project(":connect:transforms")
+    compile project(':connect:json')
```

Comment:
Hmm, should this be `compileOnly` instead (and actually same for `:connect:transforms`)? We would still prefer to load these via plugin path if possible, right?